### PR TITLE
Add $WINK for Kujira

### PR DESCRIPTION
**Pull request by codehans**

### CODE REVIEW

File: kujira/assetlist.json [hunk 1]
```diff
@@ -87,6 +87,27 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "wink",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira12cjjeytrqcj25uv349thltcygnp9k0kukpct0e/uwink",
+      "name": "WINK",
+      "display": "WINK",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:105]: please change to 'wink' in lower case

File: kujira/assetlist.json [hunk 2]
```diff
@@ -87,6 +87,27 @@
...
+      "symbol": "WINK",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/wink.svg",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:108]: can we reduce (best eliminate) the padding on the svg file? as it would make the token appear very small on displays

**Comment by codehans** [comment]
@JeremyParish69 thanks, comments addressed and PR updated

**Comment by dynalz** [comment]
Pulsar needs this!

**Pull request merged**